config.toml

```diff
@@ -1,4 +1,5 @@
 # List of files that can be appended in the track repos
 appendable_files = [
-    "CODE_OF_CONDUCT.md"
+    "CODE_OF_CONDUCT.md",
+    ".github/labels.yml
```

Comment:
@SaschaMann Is this how the path should be defined, using forward slashes?

Reply:
Yes, that should work (haven't tested but don't see why not).

Perhaps add a trailing comma so the next diff is cleaner?

Reply:
Done